pydis_site/apps/api/models/bot/nomination.py

```diff
@@ -52,3 +45,33 @@ class Meta:
         """Set the ordering of nominations to most recent first."""
 
         ordering = ("-inserted_at",)
+
+
+class NominationEntry(ModelReprMixin, models.Model):
+    """A nomination entry created by single staff."""
+
+    nomination = models.ForeignKey(
+        Nomination,
+        on_delete=models.CASCADE,
+        help_text="Nomination to what this entry belongs."
+    )
+    actor = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        help_text="The staff member that nominated this user.",
+        related_name='nomination_set'
+    )
+    reason = models.TextField(
+        help_text="Why the actor nominated this user.",
+        null=True,
+        blank=True
+    )
+    inserted_at = models.DateTimeField(
+        auto_now_add=True,
+        help_text="The creation date of this nomination entry."
+    )
+
+    class Meta:
+        """Meta options for NominationEntry model."""
+
+        verbose_name_plural = "nomination entries"
```

Comment:
It could be useful to specify a default [`ordering`](https://docs.djangoproject.com/en/3.1/ref/models/options/#ordering) here that uses the `inserted_at` date, either ascending or descending. This makes it so that nominations can be displayed in chronological order (or reverse chronological order) without having to specify anything anywhere else.